lib/lint.js

```diff
@@ -53,7 +53,7 @@ const htmlLint = () => {
 
 module.exports = ({ args }) => {
   const cli = new CLIEngine({
-    useEslintrc: false,
+    useEslintrc: true,
```

Comment:
How do the rules get merged with the rules below?

Reply:
cli rules override all other rules

Reply:
https://eslint.org/docs/user-guide/configuring#configuration-cascading-and-hierarchy

Reply:
So you can only add rules that don't conflict with ace's settings?

Reply:
yes

Reply:
inline rules override cli rules: 
![image](https://user-images.githubusercontent.com/37838624/59121791-11fc7b80-8927-11e9-84f8-bcd599c6a7d1.png)